catalog/spatial/spatial-app/pom.xml

```diff
@@ -126,6 +126,16 @@
             <artifactId>spatial-wfs-v1_0_0-schema-webapp</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.codice.ddf.spatial</groupId>
+            <artifactId>spatial-wfs-v1_1_0-source</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.codice.ddf.spatial</groupId>
+            <artifactId>spatial-wfs-v1_1_0-schema-webapp</artifactId>
```

Comment:
❗️ @glenhein I don't see the `spatial-wfs-v1_1_0-schema-webap` artifact anywhere in this PR.  Remove?